cerberus/tests/test_assorted.py

```diff
@@ -1,45 +1,10 @@
-# -*- coding: utf-8 -*-
-
 from decimal import Decimal
-from pkg_resources import Distribution, DistributionNotFound
 
 from pytest import mark
 
-from cerberus import TypeDefinition, Validator
+from cerberus import validator_factory, TypeDefinition, Validator
+from cerberus.base import UnconcernedValidator
 from cerberus.tests import assert_fail, assert_success
-from cerberus.utils import validator_factory
```

Comment:
except for tests that target deprecated features, none shall be removed.

Reply:
i meant here: https://github.com/pyeve/cerberus/pull/479/files/097fdf3570ab99120d6c6a52a505926a6752f52b#diff-9fea6a85829cee90b14b3752603ce042L23

Reply:
Could you explicitly point each one?

Reply:
not too soon. i would go though all changes in the test files and check whether a removal is also present in #431.

Reply:
Thank you for your effort and support

Reply:
excuse me, if i'm wrong reading this as 'sarcastic' comment. if i'm not wrong, please consider that all the work here happens in our free time. i appreciate that you took the initiative to update the development here, especially as it requires some effort. i think my last hint is helpful how you could proceed by yourself.

Reply:
I didn't meant any 'sarcastic' meanings here. I'm trying to go the same way as you directed, but it's not an easy thing, as you know by yourself. I asked for a help and know that it requires some effort from both sides to fix and review everything, including documentation examples.